weave/trace_server/trace_server_common.py

```diff
@@ -0,0 +1,49 @@
+import copy
+from collections import OrderedDict
+from typing import Any, Dict
+
+
+def get_nested_key(d: Dict[str, Any], col: str) -> Any:
```

Comment:
I'm kind of confused by this function, maybe some unit tests or more docstring would help. E.g. 
`get_nested_key({"a": {"b": {"c": "d"}}}, "foo")` seems like it would give you `{}`, why is that a different case than "a.b.c.e"?

Reply:
Fixed, and added tests.